### PR TITLE
provide `GOVUK_APP_DOMAIN` env as raven context

### PR DESCRIPTION
**Pull request by macool**

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Jlv2d1FT&#x2F;154-tariff16-add-env-govuk-app-domain-to-raven-getsentry-error-tracking"> [TARIFF16] Add ENV[&quot;GOVUK_APP_DOMAIN&quot;] to raven&#x2F;getsentry error tracking</a>

**Comment by matthewford** [comment]
@macool can we use server_name instead?

**Comment by matthewford** [comment]
or tags I'm not sure if we can filter by this field, tags we can though.

File: config/environments/production.rb [hunk 1]
```diff
@@ -84,4 +84,8 @@
 
   # Host for Trade Tariff API endpoint
   config.api_host = Plek.new.find("tariff-api")
+
+  config.after_initialize do
+    Raven.tags_context server_name: ENV["GOVUK_APP_DOMAIN"]
```
Comment (matthewford) [inline config/environments/production.rb]: This should be moved to the raven.rb file, and can be added as:

  config.tags = { server_name: ENV["GOVUK_APP_DOMAIN"] }

Comment (matthewford) [inline config/environments/production.rb]: I see. we're not using one, can we create one? there will be other settings we will need to tweak.

**Pull request merged**